in_text/text_03_config.yml

```
@@ -103,25 +103,25 @@ entities:
     -
       attr-label: max_layer_thick
       attr-def: >-
-        Maximum thickness of a layer
+        Maximum thickness of a layer 
       attr-defs: >-
         https://aed.see.uwa.edu.au/research/models/GLM/configuration.html
       data-min: 1.5
       data-max: 1.5
-      data-units: meters X
+      data-units: meters X 
     -
       attr-label: min_layer_thick
       attr-def: >-
-        Minimum thickness of a layer
+        Minimum thickness of a layer 
       attr-defs: >-
         https://aed.see.uwa.edu.au/research/models/GLM/configuration.html
       data-min: 0.2
       data-max: 0.2
-      data-units: meters X
+      data-units: meters X 
```

what does X represent here?
```suggestion
      data-units: meters X
```